jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/JaxRsRequestHandler.java

```diff
@@ -18,30 +18,34 @@
 @Singleton
 public class JaxRsRequestHandler implements RequestHandler<ByteBuf, ByteBuf> {
 
-    private JaxRsResources resources;
-
+    private JaxRsResources   resources;
     private ExceptionHandler exceptionHandler;
+    private ByteBufCollector collector;
 
     @Inject
     public JaxRsRequestHandler(JaxRsResourcesProvider services,
-        JaxRsResourceFactory jaxRsResourceFactory,
-        ExceptionHandler exceptionHandler
+                               JaxRsResourceFactory jaxRsResourceFactory,
+                               ExceptionHandler exceptionHandler,
+                               ByteBufCollector collector
     ) {
         this(services.getResources(),
             jaxRsResourceFactory,
             exceptionHandler,
+            collector,
             null);
     }
 
     public JaxRsRequestHandler(Object... services) {
-        this(services, new JaxRsResourceFactory(), new ExceptionHandler(), null);
+        this(services, new JaxRsResourceFactory(), new ExceptionHandler(), new ByteBufCollector(10*1024*1024), null);
```

Comment:
Perhaps make a default constructor in ByteBufCollector that makes it 10Mb, so you can remove all thos 10*1024*1024 spread in the code?

Reply:
Yeah, that would be much cleaner.